addons/sitting/XEH_preStart.sqf

```diff
@@ -1,3 +1,8 @@
 #include "script_component.hpp"
 
 #include "XEH_PREP.hpp"
+
+private _seats = QUOTE(getNumber (_x >> QQXGVAR(canSit)) == 1) configClasses (configFile >> "CfgVehicles");
+_seats = _seats apply {configName _x};
+
+uiNamespace setVariable [QGVAR(seats), _seats];
```

Comment:
```suggestion
uiNamespace setVariable [QGVAR(seats), compileFinal (_seats createHashMapFromArray [])];
```

Reply:
I initially had this, but decided against it to allow changes at mission runtime. Now, taking a second look at it, I'm not so sure anymore. Do you think it's of any interest to have something editable?